src/app/ipv/middleware.js

```diff
@@ -64,8 +64,13 @@ async function fetchUserDetails(req) {
   return generateUserDetails(userDetailsResponse, req.i18n);
 }
 
-async function handleJourneyResponse(req, res, action) {
+async function handleJourneyResponse(req, res, action, currentPage = "") {
+  if (currentPage) {
+    action += `?currentPage=${currentPage}`;
```

Comment:
It'd be preferable to use axios to set the query params - that way it'll handle stuff like escaping and building the query string for you.

i.e. adding `{ params: { currentPage } }` to the axios config for the request.